File: CodingTest/Python/solution11.py
# https://school.programmers.co.kr/learn/courses/30/lessons/42584
def solution(prices):
    answer = []
    for i in range(len(prices)):
        time = 0
        for j in range(i + 1, len(prices)):
            if prices[i] > prices[j]:
                time += 1
                answer.append(time)
                break
            else:
                time += 1
        else:
            answer.append(time)
    return answer

File: CodingTest/Java/Solution11.java
// https://school.programmers.co.kr/learn/courses/30/lessons/12945
package CodingTest.Java;

class Solution11 {
    public int solution(int n) {
        int[] fibb = new int[100001];
        fibb[0] = 0;
        fibb[1] = 1;
        for (int i = 2; i < 100001; i++) {
            fibb[i] = (fibb[i - 2] + fibb[i - 1]);
        }

        return fibb[n] % 1234567;
    }
}